src/core_atmosphere/physics/physics_wrf/module_cu_ntiedtke.F

```diff
@@ -330,17 +330,20 @@ subroutine cu_ntiedtke(                                    &
 !-------other local variables----
       integer                      :: zz, pp
 !-----------------------------------------------------------------------
+!$acc data create(rcs,rn,evap,heatflux,dx2d, &
+!$acc slimsk,prsi,ghti,zi,dot,prsl,q1,q2,q3,q1b,t1b,q11,q12,t1,u1,v1,zl, &
+!$acc omg,ghtl,kbot,ktop)
 !
 !
 !***  check to see if this is a convection timestep
 !
 
 !-----------------------------------------------------------------------
-      do j=jts,jte
-         do i=its,ite
-            cu_act_flag(i,j)=.true.
-         enddo
-      enddo
+!!!   do j=jts,jte
```

Comment:
@cponder instead of commenting the deleted code in V7. I think we should delete it as well